x-pack/plugins/security_solution/public/common/components/matrix_histogram/index.test.tsx

```diff
@@ -248,4 +256,25 @@ describe('Matrix Histogram Component', () => {
       expect(mockUseMatrix.mock.calls[0][0].skip).toEqual(true);
     });
   });
+
+  describe('useIsExperimentalFeatureEnabled = true', () => {
```

Comment:
Would you be okay changing this to be: `when the isChartEmbeddablesEnabled experimental feature flag is enabled` or `when useIsExperimentalFeatureEnabled returns true`?

As an aside, it might be helpful to create a mock like this that allows us to mock based on the argument passed in rather than the entire function in case other flags are used in the same component  

```
const getMockUseIsExperimentalFeatureEnabled = (mockMapping) => {
   return (flag) => {
      return  mockMapping.hasOwnProperty(flag) ? mockMapping[flag] : realAllowedExperimentalValues[flag]
   }
}

const mockMapping = { isChartEmbeddablesEnabled: true, ...whateverOtherFlags }
```